test/e2e/set/scyllacluster/scyllacluster_rebootstrap.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2021 ScyllaDB
```

Comment:
pls make this a separate PR, based on this one

Reply:
Opened #714